bugsnag-android-core/src/main/java/com/bugsnag/android/BreadcrumbInternal.kt

```diff
@@ -32,4 +33,13 @@ internal class BreadcrumbInternal internal constructor(
         writer.value(metadata, true)
         writer.endObject()
     }
+
+    internal fun trimMetadataStringsTo(maxStringLength: Int): Pair<Int, Int> {
```

Comment:
This sort of `Pair` is used in a few places, is it possibly worth adding a new `internal data class` for these metrics so that it's clearer? Something like:
```kotlin
internal data class TrimMetrics(
   val itemsTrimmed: Int, // breadcrumbs, strings, whatever
   val dataTrimmed: Int // chars, bytes, whatever
)
```